mobile/build.gradle.kts

```diff
@@ -38,7 +38,10 @@ android {
 
   composeOptions { kotlinCompilerExtensionVersion = libs.versions.compose.get() }
   packagingOptions { resources.excludes.add("META-INF/*") }
-  buildFeatures { compose = true }
+  buildFeatures {
+    viewBinding = true
```

Comment:
Question : do we really need `viewBinding` ?

Reply:
No, it was something that I forget to remove